libs/localization/src/lib/en.json

```diff
@@ -1279,6 +1279,25 @@
       "featureImportance": "FEATURE IMPORTANCE"
     }
   },
+  "InterpretVision": {
+    "Dashboard": {
+      "columnOne": "Title",
+      "columnTwo": "Index",
+      "columnThree": "True Y",
+      "columnFour": "Predicted Y",
+      "columnFive": "Other metadata",
+      "filter": "Filter",
+      "helperText": "Create dataset cohorts to analyze dataset statistics along filters such as predicted outcome, dataset features and error groups. Learn about the over/under presentation in your dataset.",
```

Comment:
"Learn about the over/under presentation in your dataset."
Hmm should this be representation or presentation?  Not sure.  My intuition is "representation" is the more correct term, or am I mistaken here?

Reply:
this helper text was removed from the Figma, so I'll go ahead and remove it here too